plugins/chain_api_plugin/chain_api_plugin.cpp

```diff
@@ -41,8 +41,9 @@ parse_params<chain_apis::read_only::get_transaction_status_params, http_params_t
    }
 }
 
-#define CALL_WITH_400(api_name, api_handle, api_namespace, call_name, http_response_code, params_type) \
+#define CALL_WITH_400(api_name, category, api_handle, api_namespace, call_name, http_response_code, params_type) \
```

Comment:
Seems like it should be error to configure a `http-category-address` category for a plugin that is not enabled. For example: `--http-category-address "net_ro,./x"` should be an error if `eosio::net_api_plugin` not enabled.